### ISSUE TITLE
Question About manualBiomeMappings

### ISSUE THREAD

**Issue by DonMegel** [McJty/LostCities#107]
What is the correct way to use this? I assume it is to clump biomes together the way you want but when I tried this it crashed:

```
    # Use in combination with 'allowedBiomeFactors' to manually map some biomes to others. This is a list of the format oldbiome=newbiome [default: ]
    S:manualBiomeMappings <
ominous_woods=marsh
hell=ominous_woods
rainforest=jungle_edge
jungle_hills=jungle
jungle=rainforest
jungle_edge=jungle_hills
crag=hell
wasteland=crag
bayou=dead_swamp
dead_swamp=lush_swamp
lush_swamp=quagmire
quagmire=swampland
swampland=wetland
wetland=land_of_lakes
beaches=ocean
ocean=deep_ocean
deep_ocean=volcanic_island
volcanic_island=coral_reef
coral_reef=kelp_forest
stone_beach=beaches
gravel_beach=Stone_beach
river=gravel_beach
>
```

**Comment by McJty** [comment]
The formatting is rather picky. I think you have to add indentation to those lines

**Comment by DonMegel** [comment]
Good call, that fixed it. Can I make multiple entries for the same biome? For example

wetland=land_of_lakes
wetland=jungle
wetland=lush_swamp

**Comment by McJty** [comment]
No that will not work

**Comment by DonMegel** [comment]
so each biome can only be on each side once? Like this
ominous_woods=marsh
hell=ominous_woods
rainforest=jungle_edge
jungle_hills=jungle
jungle=rainforest
jungle_edge=jungle_hills
crag=hell
wasteland=crag
bayou=dead_swamp
dead_swamp=lush_swamp
lush_swamp=quagmire
quagmire=swampland
swampland=wetland
wetland=land_of_lakes
beaches=ocean
ocean=deep_ocean
deep_ocean=volcanic_island
volcanic_island=coral_reef
coral_reef=kelp_forest
stone_beach=beaches
gravel_beach=Stone_beach
river=gravel_beach

**Comment by McJty** [comment]
No, on the right side you can repeat biomes. Just not on the left side

**Comment by DonMegel** [comment]
Ohhhhh, so this

land_of_lakes=wetland
jungle=wetland
lush_swamp=wetland
